playbooks/server-install.yml

```diff
@@ -286,24 +286,46 @@
         dest: /etc/apt/keyrings/docker.asc
         mode: 0644
 
+    - name: Add Tailscale GPG apt Key
+      get_url:
+        url: https://pkgs.tailscale.com/stable/debian/bookworm.noarmor.gpg
```

Comment:
should bookworm be: {{ ansible_lsb.codename }} ? similar to how you have it on line 297?